runtime/doc/lua-guide.txt

```diff
@@ -0,0 +1,149 @@
+*lua-guide.txt*	Nvim
+
+			    NVIM REFERENCE MANUAL
+
+			  Guide to using Lua in Nvim
+
+
+This chapter will give you the basics of using Lua in Neovim.
+
+				       Type |gO| to see the table of contents.
+
+==============================================================================
+Introduction							     *lua-guide*
+
+COMMON QUESTIONS
+
+Maybe this section isn't needed. We'll see.
+
+- Is vimscript being deprecated?
+
+Nope. Vimscript is being promoted. Vim9script however is deprecated.
+
+- Is my configuration gonna be faster?
+
+Probably not.
+
+- Explain difference between api vs lua api.
+ api for long-term guarantees it'll work, lua api for convenience. Plugins
+ should probably use api and configs should probably use lua api
+
+Probably way more, blanking out at the moment.
+
+==============================================================================
+Using Lua from Neovim
+
+You can use `:lua` to run lua code. >
```

Comment:
Lua is a proper noun and should be capitalized anywhere it's used in prose (this is inconsistent throughout the docs, but for new docs we can be good little grammarians 😊).

```suggestion
You can use `:lua` to run Lua code. >
```